OsmAnd/src/net/osmand/plus/download/IndexItem.java

```diff
@@ -32,7 +32,8 @@ public class IndexItem extends DownloadItem implements Comparable<IndexItem> {
 	long contentSize;
 	long containerSize;
 	boolean extra;
-	
+	boolean hidden;
```

Comment:
If we use this parameter only for custom maps, probably we could mobe it to CustomIndexItem